source/Module/STIG/Functions.Checklist.ps1

```diff
@@ -608,6 +667,11 @@ function Get-TargetNodeFromMof
     $TargetNode = $TargetNodeSearch.matches.value
     return $TargetNode
 }
+<#
+    .SYNOPSIS
+        Determines the type of node address
+
+#>
```

Comment:
Lets do (Mandatory = $true) instead of just (Mandatory) in the parameter blocks through the code.

Reply:
Agreed. More explicit. Done.